src/FileService.Data/FileRepository.cs

```diff
@@ -23,34 +23,31 @@ public FileRepository(IDataProvider provider, IHttpContextAccessor httpContextAc
       _httpContextAccessor = httpContextAccessor;
     }
 
-    public async Task<bool> CreateAsync(List<DbFile> files)
+    public async Task CreateAsync(List<DbFile> files)
```

Comment:
U can remove async/await from this method at all, because there is no need in context switch, u don't return anything here and u will await this method in parent method

Reply:
not sure u rly understand ))
u can ask Maria to ping me in skype to explain this )